.github/workflows/cd.yml

```diff
@@ -96,6 +96,7 @@ jobs:
           --source=.
           --set-env-vars='
           NODE_ENV=production,
+          PORT=3000,
```

Comment:
this cannot be part of the problem or the solution because the port is defined in cloud run

Reply:
then I dont understand in the logs why it says

> {"hostname":"localhost", "level":30, "msg":"Server listening at http://0.0.0.0:36749", "pid":16, "time":1.662972495749E12}
It seems starting on a random port every time

Reply:
if you start it locally with `PORT=12345 npm run dev` what happens?

Reply:
it actually starts on PORT 12345